modelforge/tests/helper_functions.py

```diff
@@ -38,8 +38,10 @@ def setup_simple_model(
         Initialized model.
     """
     from modelforge.potential import CosineCutoff, GaussianRBF
+    from modelforge.potential.utils import SlicedEmbedding
 
-    embedding = torch.nn.Embedding(max_atomic_number, nr_atom_basis)
+    embedding = SlicedEmbedding(max_atomic_number, nr_atom_basis, sliced_dim=0)
+    assert embedding.embedding_dim == nr_atom_basis
```

Comment:
There doesn't seem to really be a test of the functionality of the `SlicedEmbedding` class.